### PR TITLE
Revert "INFINITY-1975 Tweaks to Cassandra endpoints: rename node=>nat…

### PR DESCRIPTION
**Pull request by benclarkwood**
…ive-client, remove VIP, show thrift-client when enabled (#1503) (#1512)"

This reverts commit 168401552315b2b908cbcacf4d600460a498344f.

### CODE REVIEW

File: frameworks/cassandra/src/main/dist/svc.yml [hunk 1]
```diff
@@ -32,14 +32,14 @@ pods:
...
-          native-client:
+          node:
             port: {{TASKCFG_ALL_CASSANDRA_NATIVE_TRANSPORT_PORT}}
             advertise: true
-          {{#TASKCFG_ALL_CASSANDRA_START_RPC}}
-          thrift-client:
+            vip:
+              prefix: node
+              port: 9042
+          rpc:
```
Comment (nickbp) [inline frameworks/cassandra/src/main/dist/svc.yml]: this endpoint will only be available if START_RPC is enabled. its arguably a bug to display it when the feature is disabled

File: frameworks/cassandra/universe/config.json [hunk 2]
```diff
@@ -285,7 +285,7 @@
         },
         "start_rpc": {
           "type": "boolean",
-          "description": "If true Thrift RPC is enabled. This is deprecated but may be necessary for legacy applications.",
+          "description": "If true Thrift RPC is enable. This is deprecated by may be necessary for legacy applications.",
```
Comment (nickbp) [inline frameworks/cassandra/universe/config.json]: I think this can stay as-is

File: frameworks/cassandra/src/main/dist/svc.yml (hunk 1 above)
Comment (benclarkwood) [reply frameworks/cassandra/src/main/dist/svc.yml]: Yep. @nickbp can you create a ticket to make just this fix in 0.30.X?

**Comment by elezar** [comment]
Sorry for getting on this so late. I know that DSE also recently had similar changes made to the endpoint names. Should these changes be made there too?

**Comment by benclarkwood** [comment]
@elezar Have those changes already leaked out? If they have, let's not change them back.

**Comment by elezar** [comment]
From what I can tell, the changes were merged as part of mesosphere/dse-private#218.

**Pull request merged**